### ISSUE TITLE
[BUG] - Remove job after last execution not working 

### ISSUE THREAD

**Issue by xinydev** [go-co-op/gocron#111]
### Describe the bug

Remove job after last execution not works

### To Reproduce
``` go
 // Remove a Job after its last execution
    j, _ := s2.Every(1).StartAt(time.Now().Add(30*time.Second)).Do(task)
    j.LimitRunsTo(1)
    j.RemoveAfterLastRun()
```

The limit not works and the job will run forever
### Version

v0.5.1
### Expected behavior

job runs only once

**Comment by JohnRoesler** [comment]
Ah - you've exposed a different bug here, which is that no unit is set like `.Seconds()` which causes some odd behavior!

**Comment by xinydev** [comment]
@JohnRoesler  still has this bug....

**Comment by JohnRoesler** [comment]
@xinydev thanks for reporting - yes I see it still does. I am digging into this!

**Comment by JohnRoesler** [comment]
@xinydev it should be fixed now! Please let me know if you see any issue still.

**Comment by JohnRoesler** [comment]
This is released in [v0.6.0](https://github.com/go-co-op/gocron/releases/tag/v0.6.0)!
